bot/cogs/verification.py

```diff
@@ -32,26 +59,325 @@
 <#{constants.Channels.bot_commands}>.
 """
 
-BOT_MESSAGE_DELETE_DELAY = 10
+# Sent via DMs to users kicked for failing to verify
+KICKED_MESSAGE = f"""
+Hi! You have been automatically kicked from Python Discord as you have failed to accept our rules \
+within `{KICKED_AFTER}` days. If this was an accident, please feel free to join again.
+"""
+
+# Sent periodically in the verification channel
+REMINDER_MESSAGE = f"""
+<@&{constants.Roles.unverified}>
+
+Welcome to Python Discord! Please read the documents mentioned above and type `!accept` to gain permissions \
+to send messages in the community!
+
+You will be kicked if you don't verify within `{KICKED_AFTER}` days.
+"""
+
+REMINDER_FREQUENCY = 28  # Hours to wait between sending `REMINDER_MESSAGE`
+
+MENTION_CORE_DEVS = discord.AllowedMentions(
+    everyone=False, roles=[discord.Object(constants.Roles.core_developers)]
+)
+MENTION_UNVERIFIED = discord.AllowedMentions(
+    everyone=False, roles=[discord.Object(constants.Roles.unverified)]
+)
```

Comment:
I think it's probably ok to have these as module-level constants. Didn't want to pollute the actual logic with this much boilerplate code.